src/modules/logger/logger.cpp

```diff
@@ -646,9 +626,10 @@ void Logger::run()
 	int next_subscribe_topic_index = -1; // this is used to distribute the checks over time
 
 	while (!should_exit()) {
+		update_params(false);
```

Comment:
Can you move this to https://github.com/PX4/Firmware/blob/34077d8517e4581558d0b9af783f3447ebc3086f/src/modules/logger/logger.cpp#L851, to reduce delay between polling and topic readout?

Reply:
done